content/en/Platform Deep Dive/Pentests/Pentest Process/_index.md

```diff
@@ -3,10 +3,21 @@ title: "Pentest Process"
 linkTitle: "Pentest Process"
 weight: 20
 description: >
-  Learn how Cobalt tests your assets.
+  Learn about the pentest lifecycle.
 ---
 
 {{% pageinfo %}}
-Cobalt defines a sequence of events which use specific methodologies based on your asset.
+Explore the stages of the Cobalt Pentest as a Service (PtaaS) program.
 {{% /pageinfo %}}
 
+![Lifecycle of the Cobalt pentest program](/deepdive/PentestProcessFlow.png "Lifecycle of the Cobalt pentest program")
+<br><br>
+
+| | Stage | Description |
+|---|---|---|
+| 1 | **Discover** | Prepare for the pentest engagement.<ul><li>Map the attack surface of your software.</li><li>Create an account on the Cobalt platform following our invitation.</li></li>  |
+| 2 | **Plan** | Plan, scope, and schedule your pentest. To understand your expectations, we'll arrange a call with you.<ul><li>See our [Getting Started](/getting-started/) guide to learn how to launch a pentest. If you need help, [contact us](/getting-started/#get-in-touch).</li><li>Prepare the environment for our pentesters, such as set up test credentials for them.</li><li>Alert the stakeholders in your organization about the upcoming pentest.</li><li>Once you've submitted the pentest, we'll assign pentesters based on your technology stack.</li></ul> |
+| 3 | **Test** | Pentesters test your asset using various [pentest methodologies](/platform-deep-dive/pentests/pentest-process/methodologies/) and techniques.<ul><li>Pentesters share vulnerabilities that they discover in real time, in [Slack](/platform-deep-dive/collaboration/collaborate-on-pentests/#use-slack-for-communication) or in the Cobalt app. Ensure that at least one member of your team can answer questions or assist with troubleshooting. Learn more about [collaborating on a pentest](/platform-deep-dive/collaboration/collaborate-on-pentests/).</li><li>At this stage, you can start [remediating findings](/platform-deep-dive/pentests/findings/remediate-findings/) that pentesters discover.</li><li>The standard testing period is 14 days. It may vary depending on the [pentest scope](/getting-started/planning/#scope-the-pentest) and other factors.</li></ul>|
```

Comment:
"Pentesters share vulnerabilities that they discover in real time, in [Slack](/platform-deep-dive/collaboration/collaborate-on-pentests/#use-slack-for-communication) **and** in the Cobalt app."

Reply:
"Ensure that at least one member of your team can answer questions or assist with troubleshooting." Not sure what this means.